src/Message/Launch/Validator/AbstractLaunchValidator.php

```diff
@@ -51,6 +51,12 @@ abstract class AbstractLaunchValidator implements LaunchValidatorInterface
     /** @var string[] */
     protected $successes = [];
 
+    /** @var bool */
+    protected bool $isStateValidationRequired = true;
+
+    /** @var bool */
+    protected bool $isNonceValidationRequired = true;
```

Comment:
Why would you not validate nonce?

Reply:
The link provided by the portal is different. The nonce is not provided and the state can not be used. I agreed with @pedromarceOAT to validate the token only.

Reply:
@andreluizmachado You should be doing full validation, we might have misunderstood between us.
Let's align to make sure we don't open security issues.

Reply:
As far as I understood, in the normal flow, the state is created by the tool and validated on the latest endpoint to see if the state was already used or not. Here we can not do it because the state was not created by tool, but by auth server

Reply:
auth-serve.docker.localhost/lti1p3/oidc/initiation is returning a form request without the nonce parameter.

Reply:
Agreed to disable these validations for now as that information is not being generated by the Authoring app (tool).

Reply:
Should we create a ticket to not forget to return and investigate safety of the solution?

Reply:
I do not believe so as there is no plan to change it for internal calls when the portal is doing it. Unless we want to change how portal and auth server are creating these links...